examples/openvino/stable-diffusion/train_text_to_image_qat.py

```diff
@@ -397,6 +422,13 @@ def parse_args():
         choices=["cpu", "cuda"],
         help="Whether to use EMA model and where to store the EMA model.",
     )
+    parser.add_argument(
+        "--quantization_mode",
+        type=str,
+        default="moderate",
+        choices=["moderate", "aggressive"],
+        help="Whether to use EMA model and where to store the EMA model.",
```

Comment:
Could you add some information on what component will be impacted (attention,...)

Reply:
Thank you. Updated.